lua/smh/client.lua

```diff
@@ -126,6 +127,7 @@ local function Setup()
     end);
 
     menuStreams.Settings.Input.FreezeAll(SMH.DefaultData.FreezeAll);
+	menuStreams.Settings.Input.TweenDisable(SMH.DefaultData.TweenDisable);
```

Comment:
Same here, tabs to spaces
```suggestion
    menuStreams.Settings.Input.TweenDisable(SMH.DefaultData.TweenDisable);
```